scripts/technical-debt-metrics.sh

```diff
@@ -79,7 +79,7 @@ titlesPathsAndRegexes=(
   "skipAssignedInstances flags"    "*"      "set_option tactic.skipAssignedInstances"
   "adaptation notes"               "*"      "adaptation_note"
   "disabled simpNF lints"          "*"      "nolint simpNF"
-  "disabled deprecation lints"     "*"      "set_option linter.deprecated false"
+  "disabled deprecation lints (outside of \`Deprecated\` dir)"     ":^Mathlib/Deprecated"      "set_option linter.deprecated false"
```

Comment:
I think we don't need to mention this detail in the table itself, just like we don't mention that the maxHeartBeats modifications are only counted outside of the tests folder.
```suggestion
  "disabled deprecation lints"     ":^Mathlib/Deprecated"      "set_option linter.deprecated false"
```

Reply:
I suppose we could add a footnote to the message posted by the bot if we want to be very clear about it.